dll/win32/browseui/CAutoComplete.cpp

```diff
@@ -1123,7 +1126,7 @@ CAutoComplete::Init(HWND hwndEdit, IUnknown *punkACL,
     if (m_dwOptions & ACO_FILTERPREFIXES)
         FIXME(" ACO_FILTERPREFIXES not supported\n");
     if (m_dwOptions & ACO_USETAB)
-        FIXME(" ACO_USETAB not supported\n");
+        TRACE(" ACO_USETAB\n");
```

Comment:
Oh, you can remove at some point all the "TRACE" for the flags that are implemented, since these checks were meant to just print a message only when a certain flag support was not supported.

Reply:
OK.